zero/src/bin/rpc.rs

```diff
@@ -81,6 +84,7 @@ struct Cli {
 pub(crate) async fn fetch_block_prover_inputs<ProviderT, TransportT>(
```

Comment:
This will conflict with the work in https://github.com/0xPolygonZero/zk_evm/pull/682, so you probably need to rebase on top of that.